### PR TITLE
Fix the issue that clear content when paste the same content in namespace

### PR DESCRIPTION
**Pull request by BlueSodaWater**
## What's the purpose of this PR

Fix the issue that clear content when paste the same content in namespace

## Which issue(s) this PR fixes:
Fixes https://github.com/apolloconfig/apollo/issues/4919

### CODE REVIEW

**Comment by BlueSodaWater** [comment]
Sorry, I used my company's mail in last pr, so I have to close it and reopen a new one.

**Comment by BlueSodaWater** [comment]
I have read the CLA Document and I hereby sign the CLA

File: CHANGES.md [hunk 1]
```diff
@@ -28,6 +28,7 @@ Apollo 2.2.0
 * [Bump springboot version from 2.7.9 to 2.7.11](https://github.com/apolloconfig/apollo/pull/4828)
 * [[Multi-Database Support][h2] Support run on h2](https://github.com/apolloconfig/apollo/pull/4851)
 * [Fix the issue that env special case handling is missing in some case](https://github.com/apolloconfig/apollo/pull/4887)
+* [Fix the issue that clear content when paste the same content in namespace](https://github.com/apolloconfig/apollo/pull/4922)
```
Comment (nobodyiam) [inline CHANGES.md:31]:
```suggestion
* [Fix the issue that namespace content being cleared when identical content is pasted into the namespace.](https://github.com/apolloconfig/apollo/pull/4922)
```

**Review by nobodyiam — DISMISSED** [review]
LGTM

File: apollo-portal/src/main/resources/static/scripts/directive/namespace-panel-directive.js [hunk 2]
```diff
@@ -989,16 +989,8 @@ function directive($window, $translate, toastr, AppUtil, EventManager, Permissio
...
-                        for (let i = 0; i < e[0].lines.length; i++) {
-                            if (i === 0) {
-                                text = e[0].lines[0]
-                            } else {
-                                text += '\r\n' + e[0].lines[i]
-                            }
-                        }
-                        scope.namespace.editText = text
+                    if ((e[0].action === 'insert') && (scope.namespace.hasOwnProperty("editText"))) {
+                        scope.namespace.editText = e[1].session.getValue()
```
Comment (nobodyiam) [inline apollo-portal/src/main/resources/static/scripts/directive/namespace-panel-directive.js:993]:
```suggestion
                        scope.namespace.editText = e[1].session.getValue();
```

File: CHANGES.md [hunk 3]
```diff
@@ -28,6 +28,7 @@ Apollo 2.2.0
 * [Bump springboot version from 2.7.9 to 2.7.11](https://github.com/apolloconfig/apollo/pull/4828)
 * [[Multi-Database Support][h2] Support run on h2](https://github.com/apolloconfig/apollo/pull/4851)
 * [Fix the issue that env special case handling is missing in some case](https://github.com/apolloconfig/apollo/pull/4887)
+* [Fix the issue that namespace content being cleared when identical content is pasted into the namespace.](https://github.com/apolloconfig/apollo/pull/4922)
```
Comment (nobodyiam) [inline CHANGES.md:31]:
```suggestion
* [Fix the issue that namespace content being cleared when identical content is pasted into the namespace](https://github.com/apolloconfig/apollo/pull/4922)
```

**Review by nobodyiam — APPROVED** [review]
LGTM

**Pull request merged**